packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -131,6 +103,10 @@ class _TimelineFlameChartState
         curve: defaultCurve,
       );
 
+      // Maybe bail early if multiple frame selections were triggered in
```

Comment:
remove this comment. you are already bailing early enough. the existing vertical animation is already going to get cancelled so you are good. 
what might be a good idea is to before the setState return `if (_selectedFrame == selectedFrame)
`
as no animation is needed in that case. That would avoid the case where you could try to re-trigger an animation while the same animation is already running.